lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -779,72 +782,65 @@ sub _failed_check {
     my $hiddens = {
        check => $check->name,
 verify_check => md5_hex($check->test_query),
-    database => $request->{database}
+    database => $request->{database},
```

Comment:
If there's a style issue in the file (missing commas on last hash key/value pair), please change it in a separate commit. If there's no such style issue, why did this need to change in an otherwise unrelated change?